categories.json

```diff
@@ -129,6 +129,21 @@
             }
           ],
           "areas": "all"
+        },
+        "bakery": {
+          "name": "Bakery",
+          "osm_tags": [
+            {
+              "shop": "bakery|pastry"
+            },
+            {
+              "amenity": "vending_machine",
+              "vending": "bread"
+            }
+          ],
+          "areas": [
+            "DE"
+          ]
```

Comment:
Does this move bakeries into the food category? We would prefer to have independent (main) categories both for bakeries *and* chemists, if that's possible. :/

```suggestion
```